specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -7016,9 +7016,9 @@
         }
       ],
       "properties": {
-        "textExtractionAlgorithm": {
-          "$ref": "#/definitions/TextExtractionAlgorithm",
-          "description": "A value indicating which algorithm to use for extracting text. Default is printed."
+        "lineEnding": {
+          "$ref": "#/definitions/OcrSkillLineEnding",
+          "description": "A value indicating which line ending to use. Default is space."
```

Comment:
Nit: I'd quote "space" since it's used as a value from the enum.

Reply:
I was being consistent with other default values in there, but I agree.  :)  So I changed the ones for this method so they are all quoted.  Fixed in https://github.com/Azure/azure-rest-api-specs/pull/9604/commits/1d36c656614ae3c36a92e9c72b515a06bc787c3d.